test/Array/array_indexOf.baseline

```diff
@@ -664,6 +664,7 @@
 -1
 -1
 -1
+5

```

Comment:
This baseline is totally unreadable -- as long as you're in the area, if you have time, maybe you can rewrite this with the UnitTestFramework to use asserts and then delete the baseline file entirely.